ElasticsearchReportHelper.php

```diff
@@ -561,6 +561,110 @@ public static function permissionFilters(array $options) {
     }
   }
 
+   /**
+   * Output a selector for a record contexts which allows user to select from:
+   * All records (if permission is set)
+   * My records
+   * Permission filters
+   * Groups
+   *
+   * @link https://indicia-docs.readthedocs.io/en/latest/site-building/iform/helpers/elasticsearch-report-helper.html#elasticsearchreporthelper-recordContext
```

Comment:
Will you add this documentation?

Reply:
Yes I'll do that once everything is ship shape.